runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -363,12 +379,18 @@ public void onGameObjectChanged(GameObjectChanged event)
 		GameObject previous = event.getPrevious();
 		GameObject gameObject = event.getGameObject();
 
-		rocks.remove(previous);
+
 		if (ROCK_OBSTACLES.contains(gameObject.getId()))
 		{
+			rocks.remove(previous);
```

Comment:
I'm not sure if this really happens or not, but if it does, it isn't correct as it would not remove the rocks whenever they change into something else.

Reply:
done